nautobot/extras/forms/forms.py

```diff
@@ -333,6 +333,11 @@ class ConfigContextSchemaFilterForm(BootstrapMixin, forms.Form):
 
 class CustomFieldForm(BootstrapMixin, forms.ModelForm):
     label = forms.CharField(required=True, max_length=50, help_text="Name of the field as displayed to users.")
+    grouping = forms.CharField(
+        max_length=255,
+        required=False,
+        help_text="Human-readable grouping that this custom field belongs to",
+    )
```

Comment:
Is this redundant with the inclusion of the `grouping` field in the Meta section below? Won't the field max_length, blank, and help_text on the model's field provide these values?

Reply:
Yeah you are right. Confirmed by removing this code block and form worked as expected.